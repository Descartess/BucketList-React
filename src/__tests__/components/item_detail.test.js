import React from 'react';
import renderer from 'react-test-renderer';
import ItemDetail from '../../components/item_detail';

describe('banner snapshot ', () => {
  const props = {
    selectBucketItem: jest.fn(),
    onShowEditItem: jest.fn(),
    onShowDeleteItem: jest.fn(),
    completed: false,
  };
  it('renders correctly', () => {
    const rendered = renderer.create(
      <ItemDetail {...props} />,
    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
