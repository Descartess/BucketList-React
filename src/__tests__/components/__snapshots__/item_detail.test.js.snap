// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`banner snapshot  renders correctly 1`] = `
<button
  className="list-group-item"
  onClick={[Function]}
  type="button"
>
  <label
    className="checkbox-inline"
    style={undefined}
    title=""
  >
    <input
      checked={false}
      disabled={false}
      type="checkbox"
    />
  </label>
  <div
    className="pull-right"
  >
    <span
      className="glyphicon glyphicon-pencil"
      onClick={[Function]}
    />
    <span
      className="glyphicon glyphicon-trash"
      onClick={[Function]}
    />
  </div>
</button>
`;
